Gemfile.lock

```diff
@@ -387,7 +387,7 @@ GEM
       hydra-derivatives (~> 3.6)
       hydra-file_characterization (~> 1.0)
       hydra-pcdm (>= 0.9)
-    hyrax (3.6.0)
+    hyrax (3.4.2)
```

Comment:
why downgrading of Hyrax?

Reply:
Not much of a clue. It might be affecting the build.